src/interfaces/filing-header-interface.ts

```diff
@@ -0,0 +1,28 @@
+import { FilingStatus, FilingTypes } from '@/enums'
+
+/** A filing's header object from the API, ie, `filing.header`. */
+export interface FilingHeaderIF {
```

Comment:
This used to be Header Interface but now there's a different interface for Filing Header and Todo Header (even though both come in tasks from the API).